### ISSUE TITLE
[BUG?] Take a look at the accessibility check result violations in SB for Dropdowns

### ISSUE THREAD

**Issue by afebbraro** [sparkdesignsystem/spark-design-system#2159]
**Notes:**
This is for our Spark Package. Spark Core

SB a11y checks are reporting violations with our dropdown component. It would be good for u to check these out. I noticed this in Vanilla but it's more than likely happening in React and Angular.
<img width="1071" alt="Screen Shot 2019-10-15 at 5 40 13 PM" src="https://user-images.githubusercontent.com/2117593/66872036-ffe4b680-ef72-11e9-8498-ca78b3a99deb.png">

**Comment by ssabo1** [comment]
#2390 will implement this fix.
